README.md

```diff
@@ -6,15 +6,32 @@ SourceTracker was originally described in [Knights et al., 2011](http://www.ncbi
 If you use this package, please cite the original SourceTracker paper linked
 above pending publication of SourceTracker 2.
 
+# File Formats
+
+For descriptions of all file formats and options, please see the help
+documentation, available with the command `sourcetracker2 gibbs --help`. 
+
+In brief, this script requires a feature X sample contingency table
+(traditionally an OTU table) and sample metadata (traditionally a mapping file).
+Any feature table that can be read by the `biom` package will be acceptable
```

Comment:
Specifying a version for biom here is important (since different versions could read different files). I recommend changing this to:

*that can be read by ``biom-format >= 2.1.4 < 2.2.0`` will be ...*